src/token_metadatas_for_networks.ts

```diff
@@ -0,0 +1,78 @@
+import { NULL_ADDRESS } from './constants';
+import { ChainId } from './types';
+
+export interface TokenMetadataAndChainAddresses {
+    symbol: string;
+    decimals: number;
+    tokenAddresses: {
+        [ChainId.Mainnet]: string;
+        [ChainId.Kovan]: string;
+        [ChainId.Ganache]: string;
+    };
+}
+
+export const TokenMetadatasForChains: TokenMetadataAndChainAddresses[] = [
```

Comment:
Maybe worth expanding to all the ones we keep in instant (especially if we end up not returning a price when we don't know the decimals). Would be nice to have a package with all of these one day.

https://github.com/0xProject/0x-monorepo/blob/development/packages/instant/src/data/asset_meta_data_map.ts